lib/runBlock.js

```diff
@@ -83,7 +83,7 @@ module.exports = function (opts, cb) {
     async.eachSeries(block.transactions, processTx, cb)
 
     function processTx (tx, cb) {
-      var gasLimitIsHigherThanBlock = new BN(block.header.gasLimit).cmp(new BN(tx.gasLimit).add(gasUsed)) === -1
+      var gasLimitIsHigherThanBlock = new BN(block.header.gasLimit).lt(new BN(tx.gasLimit).add(gasUsed))
```

Comment:
It would make sense swapping the arguments for readability but would do that in a separate PR.